www-client/kristall/kristall-0.7.3~git.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="A cross-platform, multi-protocol visual browser for the small-internet"
+DESCRIPTION="Kristall is a browser without support for css/js/wasm or graphical \
+websites. It can display user-styled documents in several formats, \
+including Gemini, HTML, and Markdown provided by a server via Gemini, \
+Gopher, HTTP/HTTPS, and Finger protocols."
+HOMEPAGE="https://kristall.random-projects.net/"
+COPYRIGHT="2020 Felix Queißner"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="b684f94f1af9a19c1a6fd70d72097a13b75e1ca6"
+SOURCE_URI="https://github.com/MasterQ32/kristall/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="ad8dd1fa28702e94d8830fe4ba0908e44417396e7402ebfa974897bfcd8be7dd"
+SOURCE_FILENAME="kristall-$portVersion-tar.gz"
+SOURCE_DIR="kristall-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	kristall$secondaryArchSuffix = $portVersion
+	app:Kristall$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libiconv$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:git
```

Comment:
git should not be used in recipes.

Reply:
I only added it since during build, there were frequent failed attempts at using it. Building fails without it, oddly.